client/src/views/common/CameraSystemView.tsx

```diff
@@ -1,16 +1,17 @@
+import React from 'react';
+
 import ContentPage from 'components/common/ContentPage';
 import CameraRecordingContainer from 'components/common/camera_settings/CameraRecordingContainer';
 import CameraStatusContainer from 'components/common/camera_settings/CameraStatusContainer';
 import OverlayMessageContainer from 'components/common/camera_settings/OverlayMessageContainer';
 import OverlaySelectionContainer from 'components/common/camera_settings/OverlaySelectionContainer';
-import React from 'react';
 
 /**
  * Camera Settings page component
  *
  * @returns {React.Component} Component
  */
-export default function CameraSettingsView() {
+export default function CameraSystemView() {
```

Comment:
as above, and also removing the type declaration in jsdoc as that can interfere with the type intellisense